user/src/com/google/gwt/safehtml/shared/SafeUriString.java

```diff
@@ -75,4 +75,9 @@ public boolean equals(Object obj) {
   public int hashCode() {
     return uri.hashCode();
   }
+
+  @Override
+  public String toString() {
+    return asString();
```

Comment:
Please don't make `toString` a synonym for `asString`; rather copy the behavior from `SafeHtmlString`.

(but as said by Julien, please don't update the PR and send a patch on Gerrit instead).